spec/system/volunteers/index_spec.rb

```diff
@@ -263,6 +263,7 @@
           end
 
           find("[data-select-all-target='checkboxAll']").click
+          sleep(1)
```

Comment:
this is not a good fix and makes tests slow. Let's TODO remove this. @bcastillo32 can you open a ticket to clean this up?